index.js

```diff
@@ -11,8 +11,10 @@ var getDocument = require('get-document');
 
 module.exports = getWindow;
 
+var inNode = require('detect-node');
```

Comment:
I'd much rather leverage the `browser` field in the package.json file to determine which file to load. We could move the `needsIEFallback` logic out into a separate file, and a node-specific one that simply exports `false`.